### PR TITLE
island-ui/ chore: story maintenance

### PR DESCRIPTION
**Pull request by DavidBachmann**
# Story maintenance

## What

Figma links, fixed imports, remove old Button story, improve some stories.

## Why

✨

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

File: libs/island-ui/core/src/lib/Accordion/Accordion.stories.mdx [hunk 1]
```diff
@@ -15,6 +15,8 @@ import { Stack } from '../Stack/Stack'
 
 # Accordion
 
+[View in Figma](https://www.figma.com/file/pDczqgdlWxgn3YugWZfe1v/UI-Library-%E2%80%93-%F0%9F%96%A5%EF%B8%8F-Desktop?node-id=208%3A446)
```
Comment (jeremybarbet) [inline libs/island-ui/core/src/lib/Accordion/Accordion.stories.mdx:18]: No `const figmaLink` here?

File: libs/island-ui/core/src/lib/Button/Button.stories.tsx [hunk 2]
```diff
@@ -1,132 +0,0 @@
-import React from 'react'
```
Comment (jeremybarbet) [inline libs/island-ui/core/src/lib/Button/Button.stories.tsx:1]: I would keep the story with the deprecation message until no project relies on the component anymore

File: libs/island-ui/core/src/lib/Link/Link.stories.mdx [hunk 3]
```diff
@@ -56,3 +56,7 @@ import { ArrowLink } from './ArrowLink/ArrowLink'
     </Box>
   </Story>
 </Canvas>
+
+# Props
+
+<Props of={Link} />
```
Comment (jeremybarbet) [inline libs/island-ui/core/src/lib/Link/Link.stories.mdx:62]: Props is deprecated, you have to use `ArgsTable`

File: libs/island-ui/core/src/lib/AlertMessage/AlertMessage.tsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import * as React from 'react' // [1]
```
Comment (jeremybarbet) [inline libs/island-ui/core/src/lib/AlertMessage/AlertMessage.tsx:1]: Sad times, hope we get a fix for that soon :(

**Review by jeremybarbet — CHANGES_REQUESTED** [review]

File: libs/island-ui/core/src/lib/Button/Button.stories.tsx (hunk 2 above)
Comment (DavidBachmann) [reply libs/island-ui/core/src/lib/Button/Button.stories.tsx:1]: It's confusing to have Button and ButtonRC stories next to each other. Just have a deprecated message on the component itself and show new developers/users the correct story straight away.

**Review by jeremybarbet — APPROVED** [review]

**Pull request merged**